stable/redis-ha/README.md

```diff
@@ -47,7 +47,7 @@ The following tables lists the configurable parameters of the Redis chart and th
 
 | Parameter                        | Description                                           | Default                                                   |
 | -------------------------------- | ----------------------------------------------------- | --------------------------------------------------------- |
-| `redis_image`                    | Redis image                                           | `quay.io/smile/redis:4.0.2`                               |
+| `redis_image`                    | Redis image                                           | `quay.io/ianmaddox/redis:4.0.6`                           |
```

Comment:
Why use a custom image when there's an official upstream image? Do you add anything to that image that's not present in the custom one? Could you document this as well?

Reply:
The custom image is part of core functionality for this chart. I updated the readme to cover the requirements.